translations/monero-core_sk.ts

```diff
@@ -0,0 +1,2166 @@
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="346"/>
+        <source>16 or 64 hexadecimal characters</source>
+        <translation>16 alebo 64 hexadecimálnych znakov</translation>
+    </message>
+</context>
+<context>
+    <name>TxKey</name>
+    <message>
+        <location filename="../pages/TxKey.qml" line="90"/>
+        <source>Verify that a third party made a payment by supplying:</source>
+        <translation>Overte, či tretia strana uskutočnila platbu poskytnutím:</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="95"/>
+        <source> - the recipient address</source>
+        <translation> - adresa príjemcu</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="100"/>
+        <source> - the transaction ID</source>
+        <translation> - ID transakcie</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="105"/>
+        <source> - tajný kľúč transakcie poskytnutý odosielateľom</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="110"/>
+        <source>If a payment had several transactions then each must be checked and the results combined.</source>
+        <translation>Ak má platba niekoľko transakcií, potom je potrebné každú skontrolovať a skombinovať výsledky.</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="123"/>
+        <source>Address</source>
+        <translation>Adresa</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="130"/>
+        <source>Recipient&apos;s wallet address</source>
+        <translation>Adresa peňaženky príjemcu</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="152"/>
+        <source>Transaction ID</source>
+        <translation>ID transakcie</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="161"/>
+        <source>Paste tx ID</source>
+        <translation>Vložte ID transakcie</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="193"/>
+        <source>Paste tx key</source>
+        <translation>Vložte kľúč transakcie</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="216"/>
+        <source>Check</source>
+        <translation>Skontrolovať</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="185"/>
+        <source>Transaction key</source>
+        <translation>Kľúč transakcie</translation>
+    </message>
+</context>
+<context>
+    <name>WizardConfigure</name>
+    <message>
+        <location filename="../wizard/WizardConfigure.qml" line="79"/>
+        <source>We’re almost there - let’
```

Comment:
I suggest using 'Už sme skoto tam' instead of 'skoro tu'

Reply:
Corrected.